src/devtools/client/webconsole/components/Input/useHistory.tsx

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from "react-redux";
+import { getCommandHistory } from "../../selectors/messages";
+import clamp from "lodash/clamp";
+import { useState } from "react";
+
+export default function useHistory(setValue: (newValue: string) => void) {
```

Comment:
Let's at least name to `useCommandHistory` or `useEvalCommandHistory`, again, simply to not confuse it with navigation's/router's history.